### ISSUE TITLE
Git bash breaks the Ctrl v behavior

### ISSUE THREAD

**Issue by elias-rod** [git-for-windows/git#2747]
- [✔] I was not able to find an [open](https://github.com/git-for-windows/git/issues?q=is%3Aopen) or [closed](https://github.com/git-for-windows/git/issues?q=is%3Aclosed) issue matching what I'm seeing

### Setup

 - Which version of Git for Windows are you using? Is it 32-bit or 64-bit?

```
$ git --version --build-options

git version 2.27.0.windows.1
cpu: x86_64
built from commit: 907ab1011dce9112700498e034b974ba60f8b407
sizeof-long: 4
sizeof-size_t: 8
```

 - Which version of Windows are you running? Vista, 7, 8, 10? Is it 32-bit or 64-bit?

```
$ cmd.exe /c ver

Microsoft Windows [Version 10.0.18363.836]
(c) 2019 Microsoft Corporation. All rights reserved.
```

 - What options did you set as part of the installation? Or did you choose the
   defaults?

```
# One of the following:
> type "C:\Program Files\Git\etc\install-options.txt"
> type "C:\Program Files (x86)\Git\etc\install-options.txt"
> type "%USERPROFILE%\AppData\Local\Programs\Git\etc\install-options.txt"
$ cat /etc/install-options.txt

Editor Option: VisualStudioCode
Custom Editor Path: 
Path Option: Cmd
SSH Option: OpenSSH
Tortoise Option: false
CURL Option: OpenSSL
CRLF Option: CRLFAlways
Bash Terminal Option: ConHost
Git Pull Behavior Option: Merge
Performance Tweaks FSCache: Enabled
Use Credential Manager: Enabled
Enable Symlinks: Disabled
Enable Pseudo Console Support: Disabled
```

 - Any other interesting things about your environment that might be related
   to the issue you're seeing?

** insert your response here **

### Details

 - Which terminal/shell are you running Git from? e.g Bash/CMD/PowerShell/other

Bash

 - What commands did you run to trigger this issue? If you can provide a
   [Minimal, Complete, and Verifiable example](http://stackoverflow.com/help/mcve)
   this will help us understand the issue.

```
Ctrl V
```
 - What did you expect to occur after running these commands?

Paste what I have copied

 - What actually happened instead?

Nothing, if I try to paste again, it shows: ^V
The only way to paste is clicking the right mouse button. Isn't that weird?

 - If the problem was occurring with a specific repository, can you provide the
   URL to that repository to help us with testing?

** insert URL here **

**Comment by derrickstolee** [comment]
Unfortunately, CTRL+V doesn't work for...reasons. (I'm not exactly sure, but it's a hard problem in the Unix-emulation layer.)

Instead, you can use SHIFT+INS to paste, or right-click and select the Paste option from the drop-down menu:

![image](https://user-images.githubusercontent.com/570044/87743680-7428e080-c7b7-11ea-8852-2a638706ba39.png)

**Comment by elias-rod** [comment]
It was working in the 2.26.2 version. Can it be fixed?

**Comment by rimrul** [comment]
It wasn't. Ctrl+V never worked as paste in bash. Not Git bash or any other bash. It means `treat the next character verbatim` to bash. Anything else would be a bug.

**Comment by elias-rod** [comment]
Please believe me when I say that using Ctrl + V to paste is completely intuitive and that it was working until the last update.

**Comment by rimrul** [comment]
Please believe me that Ctrl+V really only means paste to GUI tools. Unix command line tools historically already had different meanings for it before the concept of a global clipboard existed.

I've worked with most released verions of Git for Windows since about1.7.9, I've worked with various other bashes (Cygwin, WSL, Linux, mac OS,...), even the occasional non-bash unix shell, as well as CMD and powershell. This feature has worked as you described in exactly **none** of them.

This just isn't how bash (or probably any unix shell) works.

**Comment by rimrul** [comment]
And just to confirm this, I just tested with `2.26.2` myself, and as expected, Ctrl+V does not paste.

```
$ git --version --build-options
git version 2.26.2.windows.1
cpu: x86_64
built from commit: 4267c88d0d8936ae723bba55e0576b1c4f3d4e1a
sizeof-long: 4
sizeof-size_t: 8

$ cmd.exe /c ver
Microsoft Windows [Version 6.3.9600]
(c) 2013 Microsoft Corporation. Alle Rechte vorbehalten.

$ cat /etc/install-options.txt
Editor Option: Nano
Custom Editor Path:
Path Option: BashOnly
SSH Option: OpenSSH
Tortoise Option: false
CURL Option: OpenSSL
CRLF Option: CRLFAlways
Bash Terminal Option: MinTTY
Performance Tweaks FSCache: Enabled
Use Credential Manager: Disabled
Enable Symlinks: Disabled
```

**Comment by pbodnar** [comment]
HI there, I've just upgraded from Git-2.25.0-64-bit.exe to Git-2.29.2.3-64-bit.exe and the `shift+insert` stopped working, I need to press just `insert` instead now (as found out on [SO](https://stackoverflow.com/questions/2304372/how-do-you-copy-and-paste-into-git-bash), where it can be seen that this topic is _generally pretty confusing_). In a regular Windows command line, the `shift+insert` does paste, while `insert` causes switches of insert/replace modes (and in PowerShell `insert` does nothing) - which is what most regular users expect, don't they? _For me then, this change is a hard to follow / accept inconsistency._

Note that as the OP, I'm also using `Bash Terminal Option: ConHost` (i. e. not using `MinTTY`). Should I report a separate bug, or is it still about the same stuff, like also historically discussed at #1117 (which doesn't state type of console used though)?
